profiler/options.go

```diff
@@ -292,6 +294,26 @@ func defaultConfig() (*config, error) {
 		}
 		c.maxGoroutinesWait = n
 	}
+	if ok, err := strconv.ParseBool(os.Getenv("DD_PROFILING_EXECUTION_TRACE_ENABLED")); err == nil && ok {
+		durationFromEnv := func(key string) (d time.Duration, err error) {
+			if v, ok := os.LookupEnv(key); ok {
+				return time.ParseDuration(v)
+			}
+			return
+		}
+		freq, err := durationFromEnv("DD_PROFILING_EXECUTION_TRACE_FREQUENCY")
+		if err != nil {
+			return nil, fmt.Errorf("parsing execution trace frequency: %s", err)
+		}
+		duration, err := durationFromEnv("DD_PROFILING_EXECUTION_TRACE_DURATION")
+		if err != nil {
+			return nil, fmt.Errorf("parsing execution trace duration: %s", err)
+		}
+		withExecutionTrace(executionTraceConfig{
+			Duration:  duration,
+			Frequency: freq,
+		})(&c)
```

Comment:
NIT: This seems a bit verbose and doesn't leverage existing env parsing patterns. I figures it's easier to show what I mean, so PTAL at https://github.com/DataDog/dd-trace-go/pull/1652/commits/c4c1103950f972e5eaa817aa967cafff6be5acda and merge https://github.com/DataDog/dd-trace-go/pull/1652 if you agree.